src/Tools/dotnet-counters/CounterMonitor.cs

```diff
@@ -177,9 +177,8 @@ private async Task<int> StartMonitor()
                     source.Dynamic.All += Dynamic_All;
                     source.Process();
                 }
-                catch (Exception ex)
```

Comment:
Maybe we can keep the exception information, but using `Debug.WriteLine` instead? What do you think?